dxr-common.js

```diff
@@ -37,15 +37,17 @@ async function getCoverage(revPromise, path) {
 async function applyOverlay(revPromise, path) {
   let result = await getCoverage(revPromise, path);
 
-  for (let [l, c] of Object.entries(result)) {
-    const line_no = document.getElementById(lineNoMap(l));
-    const line = document.getElementById(`line-${l}`);
-    if (c > 0) {
-      line_no.style.backgroundColor = 'greenyellow';
-      line.style.backgroundColor = 'greenyellow';
-    } else {
-      line_no.style.backgroundColor = 'tomato';
-      line.style.backgroundColor = 'tomato';
+  if (result.hasOwnProperty('data')) {
```

Comment:
Let's throw an exception `if (!result.hasOwnProperty('data'))`, so we can leave the rest of the code almost as is